fastlane/Fastfile

```diff
@@ -164,48 +175,51 @@ REPOSITORY_NAME="WordPress-Android"
     trigger_release_build(branch_to_build: "release/#{new_version}", app: options[:app])
   end
 
-#####################################################################################
-# update_appstore_strings
-# -----------------------------------------------------------------------------------
-# This lane gets the data from the txt files in the WordPress/metadata/ folder
-# and updates the .po file that is then picked by GlotPress for translations.
-# -----------------------------------------------------------------------------------
-# Usage:
-# fastlane update_appstore_strings version:<version>
-#
-# Example:
-# fastlane update_appstore_strings version:10.3
-#####################################################################################
+  #####################################################################################
+  # update_appstore_strings
+  # -----------------------------------------------------------------------------------
+  # This lane gets the data from the txt files in the WordPress/metadata/ folder
+  # and updates the .po file that is then picked by GlotPress for translations.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # fastlane update_appstore_strings version:<version>
+  #
+  # Example:
+  # fastlane update_appstore_strings version:10.3
+  #####################################################################################
   desc "Updates the PlayStoreStrings.po file"
   lane :update_appstore_strings do |options|
-    prj_folder = Dir.pwd + "/.."
+    # TODO: Adapt the list of files for Jetpack and JetpackPlayStoreStrings.po
```

Comment:
I plan to address that at some point next week, for now this is still very manual.